src/itx.rs

```diff
@@ -309,6 +309,94 @@ extern "C" {
     decl_itx_fns!(_12bpc, _avx2);
     decl_itx_fns!(_sse4);
     decl_itx_fns!(_ssse3);
-    // decl_itx_fn!(dav1d_inv_txfm_add_wht_wht_4x4_16bpc_avx2);
     decl_itx_fn!(dav1d_inv_txfm_add_wht_wht_4x4, _sse2);
 }
+
+macro_rules! inv_txfm_fn {
+    ($type1:ident, $type2:ident, $w:literal, $h:literal, $shift:literal, $has_dconly:literal) => {
+        paste::paste! {
+            // TODO(legare): Temporarily pub until init fns are deduplicated.
+            pub(crate) unsafe extern "C" fn [<inv_txfm_add_ $type1 _ $type2 _ $w x $h _c_erased>] <BD: BitDepth> (
+                mut dst: *mut DynPixel,
+                stride: ptrdiff_t,
+                coeff: *mut DynCoef,
+                eob: libc::c_int,
+                bitdepth_max: libc::c_int,
+            ) {
+                use crate::src::itx_1d::*;
```

Comment:
What's the glob needed for?

Reply:
The generated function references `david_inv_*` functions defined in `itx_1d`. I can change it to use an absolute path if you think that matters.

Reply:
If it's possible to make this `use` just import a single function, that'd be great.  It'd be much clearer what it's depending on.